app/components/institution-page/campus-program-application-modal.hbs

```diff
@@ -4,11 +4,18 @@
   </div>
 
   <AnimatedContainer>
-    {{#animated-if (not this.latestSavedGrantApplication) use=this.transition}}
-      <InstitutionPage::CampusProgramApplicationModal::EnterEmailStepContainer @institution={{@institution}} />
+    {{#animated-if (or (not this.latestSavedGrantApplication) this.forceShowEnterEmail) use=this.transition}}
+      <InstitutionPage::CampusProgramApplicationModal::EnterEmailStepContainer
+        @institution={{@institution}}
+        @prefilledEmail={{this.emailAddressToPrefill}}
```

Comment:
```suggestion
        @prefilledEmailAddress={{this.emailAddressToPrefill}}
```

Consistent - and "Email address" is the proper noun, email refers to the thing that is sent, not the address it was sent to